app/controllers/verify/phone_controller.rb

```diff
@@ -9,7 +9,10 @@ def new
     end
 
     def create
-      if step.complete
+      result = step.submit
+      analytics.track_event(Analytics::IDV_PHONE_CONFIRMATION, result.to_h)
```

Comment:
I disagree with `track_event` expecting a `FormResponse` object. Analytics expects an event name and hash of event properties. You should be able to pass a Hash directly to it, regardless of how that Hash was generated. Analytics should not be coupled to FormResponse. You should be able to test the Analytics class without having to invoke FormResponse.

In some cases, we don't want or need a `FormResponse` object for analytics, such as when we're adding event properties for a GET request.